docs/source/science/NISAR/OPERA Surface Displacement.ipynb

```diff
@@ -0,0 +1,4440 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.    
</pre><p>I suggest changing this:</p><p> </p>

<pre class="ql-syntax" spellcheck="false">with s3.open(results[0].getDownloadUrl(), "rb") as f:
&nbsp;&nbsp;  opera_data = h5py.File(f, "r")
</pre><p> </p><p>to this:</p><p> </p>

<pre class="ql-syntax" spellcheck="false">with (
    s3.open(results[0].getDownloadUrl(), "rb") as f,
    h5py.File(f, "r") as opera_data,
):
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>